src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -101,15 +101,14 @@ const ActionsPane = (props: {contentHeight: string}) => {
 
   const onSelect = useCallback(
     (absolutePath: string) => {
-      if (!selectedResourceId) {
-        return;
+      if (selectedResource) {
+        dispatch(
+          saveUnsavedResource({
+            resource: selectedResource,
+            absolutePath,
+          })
+        );
       }
-      dispatch(
-        saveUnsavedResource({
-          resourceId: selectedResourceId,
-          absolutePath,
-        })
-      );
     },
     [selectedResourceId, dispatch]
```

Comment:
You need chabge `selectedResourceId` to `selectedResource` here

Reply:
oops - will do - thanks!